web/ui/dashboard/tailwind.config.js

```diff
@@ -57,7 +57,8 @@ module.exports = {
 				80: '#31323D',
 				60: '#5F5F68',
 				40: '#737A91',
-				20: '#E8E8E9'
+				20: '#E8E8E9',
+				10: '#EDEDF5'
```

Comment:
Where is this color from?

Reply:
this is the color used for almost all borders and also used for the required tags